Testing/VelaptorTests/NativeInterop/OpenGL/OpenGLServiceTests.cs

```diff
@@ -384,12 +386,18 @@ public void LabelVertexArray_WhenInvoked_LabelsVertexArray(string label, string
     public void LabelBuffer_WithInvalidBufferType_ThrowsException()
     {
         // Arrange
-        var exceptionMsg = "Exception of type 'System.ArgumentOutOfRangeException' was thrown. ";
-        exceptionMsg += $"(Parameter 'bufferType'){Environment.NewLine}Actual value was 123.";
+        var invalidValue = 123;
+        var expected = $"The value of argument 'bufferType' ({invalidValue}) is invalid for Enum type " +
+                       $"'{nameof(OpenGLBufferType)}'. (Parameter 'bufferType')";
+
         var service = CreateService();
 
         // Act
-        var act = () => service.LabelBuffer(It.IsAny<uint>(), It.IsAny<string>(), (OpenGLBufferType)123);
+        var act = () => service.LabelBuffer(It.IsAny<uint>(), It.IsAny<string>(), (OpenGLBufferType)invalidValue);
+
+        // Act & Assert
+        act.Should().Throw<InvalidEnumArgumentException>()
+            .WithMessage(expected);
 
         // Assert
         act.Should().Throw<ArgumentOutOfRangeException>().WithMessage(exceptionMsg);
```

Comment:
Please fix compile error.

I am getting a build error because the argument being sent into the `WithMessage()` method should be named `expected`.
The variable `exceptionMsg` does not exist.

 ```suggestion
        act.Should().Throw<ArgumentOutOfRangeException>().WithMessage(expected);
```

Reply:
I think this is caused by a wrong merge.

Reply:
You are correct.  This was due to the update to this PR from a previous merge into the base branch.  Good catch.